cmd/log-validator/main.go

```diff
@@ -149,9 +155,19 @@ func main() {
 					logger.Errf("error while tailing %s: %s", t.Filename, err)
 					continue
 				}
+				const prefix = "log validator error"
+				// Do not consume our own output. This prevents runaway scenarios in
+				// case of a problem that causes all log lines to be invalid.
+				if strings.Contains(line.Text, prefix) {
```

Comment:
`prefix` won't actually be a prefix of the line, it's a prefix of the message portion of the line. Might make sense to move this check into `lineValid` though so that this can be more accurately tested (and also then we can add a unit test).